backend/api/Controllers/Models/RescheduleMissionQuery.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Api.Database.Models;
+
+namespace Api.Controllers.Models
+{
+    public class RescheduleMissionQuery
+    {
+        public MissionSourceType MissionType { get; set; }
+        public string RobotId { get; set; }
+        public DateTimeOffset DesiredStartTime { get; set; }
+        public TimeSpan? InspectionFrequency { get; set; }
```

Comment:
Should these two be a part of this? The frequency should not change every time a mission is scheduled.

We are also not using a desired start time other than now I think so might not be necessary as an input yet.

Reply:
Since we are already using DesiredStartTime in the main branch I will keep it in even if it does not have an effect, so that we don't need to worry about readding it later, but I will remove InspectionFrequency.